### PR TITLE
add descriptions for plugins

### LINKED ISSUE #459: Search autocomplete items should describe what they do
image

we can make those say something more than "plugin" and we should. work will be done here and in plugins https://github.com/GoogleChromeLabs/ProjectVisBug/blob/main/app/features/search.js#L12

### PR DESCRIPTION
**Pull request by hchiam**
to fix #459 

image

### CODE REVIEW

File: app/plugins/tag-debugger.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ export const commands = [
   'osmani',
 ]
 
+export const description = 'outline every DOM element with a random color, to visualize layout'
+
 export default async function() {
+  let i, A;
```
Comment (hchiam) [inline app/plugins/tag-debugger.js:10]: otherwise console error in Chrome

File: app/plugins/shuffle.js [hunk 2]
```diff
@@ -33,16 +35,16 @@ export default async (selectedElement) => {
...
+    for (let i = 0; i < shuffledElementsArray.length; i++) {
+      element.appendChild(shuffledElementsArray[i])
     }
   };
   const { selected } = selectedElement;
   selected.map(selectedElem => {
     const siblings = getSiblings(selectedElem);
     const shuffledSiblings = shuffle(siblings);
-    appendSuffledSiblings(selectedElem, shuffledSiblings);
+    appendShuffledSiblings(selectedElem, shuffledSiblings);
```
Comment (argyleink) [inline app/plugins/shuffle.js:48]: hehe thx 🙂

File: app/plugins/tag-debugger.js (hunk 1 above)
Comment (argyleink) [reply app/plugins/tag-debugger.js:10]: interesting.. thanks for finding it!

**Review by argyleink — APPROVED** [review]
very cool, nice work! exporting the description is a great choice. sorry you had to go update each registry item!

overall looks great, appreciate it!

**Comment by hchiam** [comment]
no worries, vscode multi cursor selection for the win! plus i got to discover a bunch of other tools/bookmarklets and hidden features :)

File: app/plugins/tag-debugger.js (hunk 1 above)
Comment (hchiam) [reply app/plugins/tag-debugger.js:10]: yeah, i noticed the plugin wasn’t working until i added this line.  now that it's slightly more discoverable, it’ll work just in time

**Pull request merged**